docs/src/main/sphinx/admin/properties-general.rst

```diff
@@ -36,3 +36,24 @@ across nodes in the cluster. It can be disabled, when it is known that the
 output data set is not skewed, in order to avoid the overhead of hashing and
 redistributing all the data across the network. This can be specified
 on a per-query basis using the ``redistribute_writes`` session property.
+
+``protocol.v1.alternate-header-name``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Type:** ``string``
+
+The 353-e release of Trino changes the HTTP client protocol headers to start
+with ``X-Trino-``. Legacy clients can be temporarily supported by setting this
```

Comment:
Legacy clients .. maybe we need to spell this out 

Clients for versions 350 and lower expect the HTTP headers to use X-PRESTO, while newer clients expect X-TRINO. You can support these older clients by setting this property to PRESTO.

Reply:
Will use case consistent with blog post and migration guide though.